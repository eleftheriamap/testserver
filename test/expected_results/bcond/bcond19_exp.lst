
test/exp/test/gen/bcond/bcond19_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	eb0b017f 	cmp	x11, x11
   4:	911e917c 	add	x28, x11, #0x7a4
   8:	eb1c017f 	cmp	x11, x28
   c:	5400004d 	b.le	0x14
  10:	9170af7b 	add	x27, x27, #0xc2b, lsl #12
  14:	911dcf7b 	add	x27, x27, #0x773
  18:	8a000000 	and	x0, x0, x0
