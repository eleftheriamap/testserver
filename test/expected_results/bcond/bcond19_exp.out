Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000773
X28    = 00000000000007a4
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 0000000000000018
PSTATE : N---
Non-zero memory:
0x00000000 : 0xeb0b017f
0x00000004 : 0x911e917c
0x00000008 : 0xeb1c017f
0x0000000c : 0x5400004d
0x00000010 : 0x9170af7b
0x00000014 : 0x911dcf7b
0x00000018 : 0x8a000000
